pages/common/meshlabserver.md

```diff
@@ -0,0 +1,23 @@
+# meshlabserver
+
+> Command line interface for the MeshLab 3D mesh processing software.
+
+- Convert an STL file to an OBJ file:
+
+`meshlabserver -i {{input.stl}} -o {{output.obj}}`
+
+- Convert a WRL file to a OFF file, inlcuding the vertex and face normals in the output:
+
+`meshlabserver -i {{input.wrl}} -o {{output.off}} -om vn fn`
+
+- Dump a list of all the available processing filters into a text file:
```

Comment:
If we are not using the `.txt` extension, maybe its just better to use "into a file" ?